tensorboard/webapp/BUILD

```diff
@@ -100,26 +100,17 @@ tf_web_library(
     ],
 )
 
-tf_web_library(
-    name = "material_theme",
-    srcs = [
-        "@npm//:node_modules/@angular/material/prebuilt-themes/indigo-pink.css",
-    ],
-    path = "/@angular",
-    strip_prefix = "node_modules/@angular/material/prebuilt-themes",
-)
-
 # Bundle the Angular app with the Polymer components and plugins, as a library.
 tf_web_library(
     name = "tensorboard-webapp",
     srcs = [
         "index.html",
+        ":styles.css",
```

Comment:
nit: sort the two dependencies alphabetically.

Reply:
actually, it is buildifier that does this (even after manual change).